pylint_odoo/misc.py

```diff
@@ -172,6 +168,63 @@ def _check_{NAME_KEY}(self, module_path)
                         node.file = node_file_original
                         node.lineno = node_lineno_original
 
+    def visit_module(self, node):
+        self.wrapper_visit_module(node)
+
+    def add_message(self, msg_id, line=None, node=None, args=None,
+                    confidence=UNDEFINED):
+        version = (self.manifest_dict.get('version')
+                   if isinstance(self.manifest_dict, dict) else '0')
+        version = LooseVersion(version)
+        short_version = '.'.join(map(str, version.version[:2]))
+        match = re.match(
+            DFTL_MANIFEST_VERSION_FORMAT.format(
+                valid_odoo_versions=short_version), version.vstring)
+        if not match:
+            valid_odoo_versions = self.linter._all_options[
+                'valid_odoo_versions'].config.valid_odoo_versions
+            version = LooseVersion(valid_odoo_versions[0] if
```

Comment:
version is not used anymore in the code below

Reply:
Your be right, I deleted the unnecessary code